apps/tup-cms/README.md

```diff
@@ -16,6 +16,9 @@ A `Makefile` has been included for convenience. You may use
 make start
 ```
 
+> **Important**
+> If you must run `python manage.py collectstatic` locally, do so via `python manage.py collectstatic --ignore assets/*/font*.css` to avoid error.
```

Comment:
Do not mention locally. Any server needs this.

```suggestion
> If you must run `python manage.py collectstatic`, do so via `python manage.py collectstatic --ignore assets/*/font*.css` to avoid error.
```